resources/views/students/edit.blade.php

```diff
@@ -2,7 +2,7 @@
   <i class="fa fa-user" aria-hidden="true"></i>
   {{$student->First . " " . $student->Last}}
 </h5>
-<form method="POST" action="{{ route('editROUTE') }}">
+<form method="POST" action="{{ route('students.update', '$student') }}">
```

Comment:
Wasn't all of this already changed in the other PR? Merge that, then rebase this branch and these changes should be removed